crates/cheatnet/tests/contracts/src/segment_arena_user.cairo

```diff
@@ -0,0 +1,9 @@
+#[starknet::contract]
+mod SegmentArenaUser {
+    #[storage]
+    struct Storage { }
+    #[external(v0)]
+    fn interface_function(ref self: ContractState){
+        let felt_dict: Felt252Dict<felt252> = Default::default();
```

Comment:
Would be probably nice to have additional operation performed on it in this test @Arcticae, are there any?

Reply:
It's using the builtin in this case anyway, which was our previous point of failure